transit_odp/avl/views/detail.py

```diff
@@ -85,11 +85,9 @@ def get_context_data(self, **kwargs):
             ).first()
```

Comment:
I wonder if it will be possible to add a unit test that compares the two?
If its too hard or will take ages I'm not into it but if its easy it could be a good way to go.

Reply:
Ok, I'll do it!